src/git.ts

```diff
@@ -3,29 +3,49 @@ import {PathLike} from 'fs';
 
 import {execWithStdoutCap} from './process-utils';
 
+/**
+ * Wrapper class for acting on a Git repository with the Git binary.
+ */
 export class Repo {
   private targetDirectory: PathLike;
 
+  /**
+   * @param targetDirectory - Any directory within the worktree of the Git
+   *    repository to work with
+   */
   constructor(targetDirectory: PathLike) {
     this.targetDirectory = targetDirectory;
   }
 
+  /**
+   * Restore all modified files in the current.
```

Comment:
I think you can skip documenting the return type as when we don't define the return type for void functions, I feel a similar logic can be extended here. Moreover, the promise after resolving would anyway return `undefined` so maybe it doesn't make sense.

I am not aware of any convention that instructs one to document `Promise<void>`.

Reply:
> I think you can skip documenting the return type as when we don't define the return type for void functions, I feel a similar logic can be extended here. Moreover, the promise after resolving would anyway return undefined so maybe it doesn't make sense.

Right, then I think it makes sense to not add any docs for them, seems rather self-explanatory. Thanks for the input.